paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -330,6 +331,18 @@ internal class CustomerSheetViewModel @Inject constructor(
             )
         } else {
             backStack.update {
+                it.lastOrNull()?.let { viewState ->
```

Comment:
Nit: Could move the mapping from view state to reporter screen to an extension property. I think we can also expect `last()` to not error out.